rules/properties-alphabetical-order/tests/index.js

```diff
@@ -68,6 +68,21 @@ testRule({
 		{
 			code: 'a { font-size: 1px; -moz-osx-font-smoothing: grayscale; -webkit-font-smoothing: antialised; font-weight: bold; }',
 		},
+		{
+			// blocked by https://github.com/hudochenkov/stylelint-order/issues/78
+			skip: true,
+			code: 'a { display: block; margin: 0 auto 5px 0; Margin: 0 auto 5px 0; width: auto; }',
+		},
+		{
+			// blocked by https://github.com/hudochenkov/stylelint-order/issues/78
+			skip: true,
+			code: 'a { display: block; Margin: 0 auto 5px 0; margin: 0 auto 5px 0; width: auto; }',
+		},
+		{
+			// blocked by https://github.com/hudochenkov/stylelint-order/issues/78
+			skip: true,
+			code: 'a { align: center; Border-width: 1px; Border-top-width: 2px; color: red; }',
```

Comment:
I would also add cases:

```
border-width: 1px; Border-top-width: 2px;
Border-width: 1px; border-top-width: 2px;
```

Reply:
![obraz](https://github.com/user-attachments/assets/5bb29e60-2200-4383-9ea8-2296a1c8ae7a)
If above properties are correct, then the shorthand comparison must be altered
https://github.com/hudochenkov/stylelint-order/blob/416d00c13c416a3529f712f3dd8cd0fbf63ea909/rules/checkAlphabeticalOrder.js#L11-L19

Reply:
That's why we need those tests :)

But I think once config values and properties names will be lower-cased before comparison, everything will just work.